compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -184,39 +185,83 @@ class UntypedFnSignature {
 
 using SubstitutionsMap = std::unordered_map<const uast::Decl*, types::QualifiedType>;
 
-struct CallInfoActual {
-  types::QualifiedType type;
-  UniqueString byName;
+/** CallInfoActual */
+class CallInfoActual {
+ private:
+  types::QualifiedType type_;
+  UniqueString byName_;
+
+ public:
+  CallInfoActual(types::QualifiedType type, UniqueString byName)
+      : type_(type), byName_(byName) {}
+
+  /** return the byName */
+  const types::QualifiedType& type() const { return type_; }
 
-  bool operator==(const CallInfoActual& other) const {
-    return type == other.type &&
-           byName == other.byName;
+  /** return the byName */
```

Comment:
When a Call is represented, if there was an argument passed with named argument passing, like in e.g., `f(number=3)`, then the `byName` here will contain the name (`number` in this case). Of course it would be great to include an example like this in the docs comment. I also think we could probably find a better name for the method but one doesn't immediately occur to me.

Reply:
Thanks!